README.md

```diff
@@ -30,111 +30,259 @@ Recommended Setup
 
 Before you can start, you have to download the browscap.ini file and convert it into a cache. There are two ways.
 
-1. Download the file and convert it in two steps. The downloaded file will be stored in a local file, but there is no check
+a. Download the file and convert it in two steps. The downloaded file will be stored in a local file, but there is no check
    if the remote file has changed. If your cache gets corrupted you only need to rerun the `convert` command.
 
-```php
-vendor/bin/browscap-php browscap:fetch
-vendor/bin/browscap-php browscap:convert
-```
+    ```php
```

Comment:
These are double-encoded now: see https://github.com/mimmi20/browscap-php/blob/39ee90c30113462d4523902c1d4d9e6f5cb8af69/README.md#recommended-setup for preview